src/index.js

```diff
@@ -0,0 +1,273 @@
+/* eslint no-use-before-define: 0, no-extra-parens: 0 */
```

Comment:
Interesting - so does that mean internally there is still hoisting going on? I wouldn't expect that to run.